pkg/pb/gc.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package gc;
+
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+service GarbageCollection {
+    rpc Delete(DeleteRequest) returns (DeleteResponse);
```

Comment:
I'm not sure `Delete` is the right word for this. Collect or Retain might be better.

Reply:
I think Retain is indeed better. I've changed it.

Reply:
I think Retain is indeed better. I've changed it.